plugin.video.rivedila7/.gitignore

```diff
@@ -0,0 +1,123 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Exclude from your  submission please.

Reply:
Yes, I and Luivit are together the owner of this plugin. You can see it in the addon.xml

Why we need remove the .gitignore file ?

Reply:
Great. The .gitignore does not belong in a Kodi plugin. So it should not be included in the PR.

Reply:
@basrieter 
Gitignore removed